waspc/data/Generator/templates/sdk/client/operations/index.ts

```diff
@@ -0,0 +1,22 @@
+// PUBLIC API
+export * from './actions'
+// MOSTLY PUBLIC API (see the file for details)
+export * from './queries'
+
+export {
+    // PUBLIC API
+    useAction,
+    // PUBLIC API
+    useQuery,
+    // PUBLIC API
+    type OptimisticUpdateDefinition,
+} from './core'
+
+export {
+    // PUBLIC API
+    configureQueryClient,
+    // PRIVATE API (framework code)
+    initializeQueryClient,
+    // PRIVATE API (framework code)
+    queryClientInitialized
```

Comment:
I would maybe then not export these two here from index.ts and let our code obtain that code directly from queryClient.

Reply:
This wouldn't quite work because of `configureQueryClient`. We agreed to take care of it later.